modules/sfm/CMakeLists.txt

```diff
@@ -0,0 +1,90 @@
+
+if (NOT HAVE_EIGEN)
+  message(STATUS "Module opencv_sfm disabled because Eigen library is required.")
+  ocv_module_disable(sfm)
+  return()
+endif()
+
+set(the_description "SFM algorithms")
+
+ocv_warnings_disable(CMAKE_CXX_FLAGS -Wundef -Winvalid-pch)
+
+if(UNIX)
+  if(CMAKE_COMPILER_IS_GNUCXX OR CV_ICC)
+     set(CMAKE_CXX_FLAGS "${CMAKE_CSS_FLAGS} -fPIC")
```

Comment:
I get this error:

t_triangulation.cpp.o
Building CXX object modules/sfm/CMakeFiles/opencv_test_sfm.dir/test/test_fundamental.cpp.o
[100%] Building CXX object modules/sfm/CMakeFiles/opencv_test_sfm.dir/test/scene.cpp.o
Linking CXX executable ../../bin/opencv_test_sfm
CMakeFiles/opencv_test_sfm.dir/test/test_triangulation.cpp.o: En la función `checkTriangulation(int, int, bool, float, float) [clone .constprop.73]':
test_triangulation.cpp:(.text._ZL18checkTriangulationiibff.constprop.73+0xce6): referencia a `cv::triangulatePoints(cv::_InputArray const&, cv::_InputArray const&, cv::_OutputArray const&)' sin definir
collect2: error: ld returned 1 exit status
make[2]: *** [bin/opencv_test_sfm] Error 1
make[1]: *** [modules/sfm/CMakeFiles/opencv_test_sfm.dir/all] Error 2
make[1]: *** Se espera a que terminen otras tareas....
Linking CXX shared library ../../lib/cv2.so
[100%] Built target opencv_python2
make: *** [all] Error 2